Solutions/FalconFriday/Analytic Rules/Disable or Modify Tools.yaml

```diff
@@ -0,0 +1,47 @@
+id: 20d52a04-b5d8-402d-88e2-7929d12cbdcd
+name: Disable or Modify Tools
+description: |
+  This detection watches the commandline logs for known commands that are used to disable the Defender AV. This is based on research performed by @olafhartong on a large sample of malware for varying purposes. 
+  Note that this detection is imperfect and is only meant to serve as basis for building a more resilient detection rule. 
+  Some software behaves weirdly, we've at least identified that JetBrains and Android Studio are behaving oddly. Not sure why, yet. Perhaps I should stop using JetBrains?
```

Comment:
may be this line can be rephrased and this section can be removed `Not sure why, yet. Perhaps I should stop using JetBrains?`. 

Blog is mentioned but link is not provided , consider adding reference link.

Reply:
Nice catch! That sentences should've never been in there 👍